odoo_sh/getting_started/branches.rst

```diff
@@ -275,9 +275,30 @@ to manually set it up again on every commit. If you choose new build for a stagi
 make a fresh copy from the production build every time a commit is pushed. A branch that is put
 back from staging to development will automatically be set to 'Do nothing'.
 
+**Modules installation**
+
+Choose the modules to install automatically for your development builds.
+
+.. image:: ./media/interface-settings-modulesinstallation.png
+   :align: center
+
+* *Install only my modules* will install the modules of the branch only. This is the default option.
+  The :ref:`submodules <odoosh-advanced-submodules>` are excluded.
+* *Full installation (all modules)* will install the modules of the branch, the modules included in the submodules
+  and all standard modules of Odoo. When running the full installation, the test suite is disabled.
+* *Install a list of modules* will install the modules specified in the input just below this option.
+  The names are the technical name of the modules, and they must be comma-separated.
+
+If the tests are enabled, the standard Odoo modules suite can take up to 1 hour.
+This setting applies to development builds only.
+Staging builds duplicate the production build and the production build only installs base.
+
+
 **Test suite**
 
 For development branches, you can choose to enable or disable the test suite. It's enabled by default.
+When the test suite is enabled, you can restrict the tests to bu run for specific tests by adding test tags.
```

Comment:
"to BE run" :D

But I'd simplify in "you can restrict them by specifying test tags"

For the line below: I'm not sure how it gives in the interface but maybe include the link in the "test tags" from the phrase above?